causal/oidc/CVE-2025-24856.yaml

```diff
@@ -0,0 +1,8 @@
+title: 'TYPO3-EXT-SA-2025-001: Account Takeover in extension "OpenID Connect Authentication" (oidc)'
+link: https://typo3.org/security/advisory/typo3-ext-sa-2025-001
+cve: CVE-2025-24856
+branches:
+    main:
+        time: 2025-01-27 18:56:00
+        versions: ['3.0.0']
```

Comment:
```suggestion
        versions: ['>=3.0.0', '<4.0.0']
```

Like this?

Reply:
There's literally only a single 3.0.0 release so kind of would have worked too ;-)